content/pt-br/docs/concepts/policy/node-resource-managers.md

```diff
@@ -0,0 +1,22 @@
+---
+reviewers:
+- derekwaynecarr
+- klueska
+title: Node Resource Managers 
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+
+Para dar suporte a latência crítica e a cargas de trabalho de alta taxa de transferência. O kubernets oferece um conjunto de grenciadores de recursos. Os gerenciadores visam coordenar e otimizar o alinhamento de recursos do nó(s) para pods configurados com um requisito específico para CPUs, dispositivos e recursos de memória (grandes páginas).
```

Comment:
typo: "transferência, o kubernetes"

Reply:
Some points to change in this paragraph:
* Typo in "grenciadores".
* It's best to keep the capital P in Pod, as it refers to a Kubernetes API object.
* Hugepages refers to a very specific concept in the Linux space. It's best to keep it as "hugepages" in the translation, but stylize as italic.

Reply:
Do I keep Pod or Pods? in the original document it is written as Pods

Reply:
Yes, it should be kept as Pods like in the original.
I also see the typo in Kubernets is still there, though (should be "Kubernetes", with a capital K as well).